src/components/Ingredient/Form/Form1.js

```diff
@@ -1,11 +1,53 @@
 import React, { Component } from "react";
 
+import Selectbox from "../Selectbox";
+import { groceristar } from "@groceristar/groceristar-fetch";
+
 class Form1 extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ingredient: "",
+      selectedDepartment: ""
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+  }
+  handleChange(e) {
+    this.setState({
+      ingredient: e.target.value
+    });
+  }
+  handleSubmit(e) {
+    let obj = {
+      ingredient: this.state.ingredient,
+      department: this.state.selectedDepartment
+    };
+    console.log(obj);
```

Comment:
please also add here a comment or maybe call an empty method, because later at this place we'll send a query to our server